packages/libs/react-ui/src/components/NavHeader/NavHeaderContent.tsx

```diff
@@ -1,10 +1,11 @@
 import { childrenClass } from './NavHeader.css';
 
-import type { FC } from 'react';
+import type { INavHeaderSelectProps } from '@components/NavHeader/NavHeaderSelect';
+import type { FC, FunctionComponentElement } from 'react';
 import React from 'react';
 
 export interface INavHeaderContentProps {
-  children: React.ReactNode;
+  children: React.ReactNode | FunctionComponentElement<INavHeaderSelectProps>[];
```

Comment:
I guess this is redundant? As we mainly use `FunctionComponentElement` to restrict but since it accepts any Node it won't actually restrict anything.